runtime/colors/vim.lua

```diff
@@ -0,0 +1,290 @@
+-- Name:         vim
+-- Description:  Vim's default color scheme
+-- Author:       Bram Moolenaar <Bram@vim.org>
+-- Maintainer:   The Vim Project <https://github.com/vim/vim>
+-- Website:      https://github.com/vim/vim
+-- License:      Same as Vim
+-- Last Updated: 2023 Aug 10
+
+-- This is Vim's default color scheme. It doesn't define the Normal
+-- highlighting, it uses whatever the colors used to be.
+
+vim.cmd('hi clear')
```

Comment:
```suggestion
vim.cmd.highlight('clear')
```
(just an alternative suggestion; should be nanoseconds faster since the command need not be parsed)